kwok/Dockerfile

```diff
@@ -4,11 +4,12 @@ ARG CLUSTER_NAME=kwok
 ENV CLUSTER_NAME=${CLUSTER_NAME}
 
 COPY --chmod=755 user-signups.sh entrypoint.sh toolchain.dev.openshift.com_usersignups.yaml ./
```

Comment:
```suggestion
COPY --chmod=755 user-signups.sh toolchain.dev.openshift.com_usersignups.yaml ./
```

Reply:
Also we need to remove kwok/entrypoint.sh as part of this change.

Reply:
done